docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx

```diff
@@ -60,7 +60,7 @@ For reference see the [CLI's docs](https://github.com/mateusz1913/vision-camera-
   CMSampleBufferRef buffer = frame.buffer;
   UIImageOrientation orientation = frame.orientation;
   // code goes here
-  return @[];
+  return @{};
```

Comment:
Maybe this should also be `null`/`nil` so it's the same on both iOS and Android? Now this returns an empty object/Dictionary...